docs/reference/graph/graph.yaml

```diff
@@ -412,76 +412,154 @@ spec:
   edges:
     - label: CE_MODULE_LOAD
       description: A container can load a kernel module on the node.
+      attck: 
+        - id: T1611
+          name: Escape to Host
     - label: CE_NSENTER
       description: >-
         Container escape via the nsenter built-in linux program that allows
         executing a binary into another namespace.
+      attck:
+        - id: T1611
+          name: Escape to Host
     - label: CE_PRIV_MOUNT
       description: >-
         Mount the host disk and gain access to the host via arbitrary filesystem
         write
+      attck:
+        - id: T1611
+          name: Escape to Host
     - label: CE_SYS_TRACE
       description: >-
         Given the requisite capabilities, abuse the legitimate OS debugging
         mechanisms to escape the container via attaching to a node process.
+      attck:
+        - id: T1611
+          name: Escape to Host
     - label: CE_UMH_CORE_PATTERN
       description: >-
         Abuse the User Mode Helper (UMH) mechanism to execute arbitrary code in
         the host.
+      attck:
+        - id: T1611
+          name: Escape to Host
     - label: CE_VAR_LOG_SYMLINK
       description: Abuse the /var/log symlink to gain access to the host filesystem.
+      attck:
+        - id: T1611
+          name: Escape to Host
     - label: EXPLOIT_HOST_READ
       description: Read sensitive files on the host.
+      attck:
+        - id: T1611
+          name: Escape to Host
     - label: EXPLOIT_HOST_WRITE
       description: Write sensitive files on the host.
+      attck:
+        - id: T1611
+          name: Escape to Host
     - label: EXPLOIT_CONTAINERD_SOCK
       description: Exploit the containerd socket to gain access to the host.
+      attck:
+        - id: TA0008
+          name: Lateral Movement
```

Comment:
Now I'm wondering if we should allow for multiple references, I would have put this as `Escape to Host` more than lateral movement 😄

Reply:
Yes, that's why I added the capability to add multiple references, as technically, we should choose the most appropriate, but sometimes you can be in the middle of 2 appropriate techniques.

Reply:
Yeah I misread the json schema and didn't see that it was already the case, thanks!